### ISSUE TITLE
User guide: consolidate prerequisites into a single section or page

### ISSUE THREAD

**Issue by chalin** [google/docsy#1180]
With the introduction of more than one [installation option](https://www.docsy.dev/docs/get-started/#installation-options), we now have duplicate explanations of how to setup prerequisites. For example, compare:

- https://www.docsy.dev/docs/get-started/docsy-as-module/installation-prerequisites/
- https://www.docsy.dev/docs/get-started/other-options/#prerequisites

For those parts that apply to all installation options (which is most if not all parts), these should be pulled out into a separate **Prerequisites** section.

WDYT @LisaFC @emckean

**Comment by LisaFC** [comment]
Yes, I think that's a great idea (in fact I literally did the same thing recently for the docs for something else).

We can add a quick intro with "These prerequisites apply to all installation options..." etc. etc. Then each installation option guide can have its own "Before you begin" with:
- Ensure you have all the prerequisites as described in...
- [any option-specific requirements]

**Comment by chalin** [comment]
This is still relevant, in fact now some sections are becoming even more out-of-sync (e.g., #1597). The content of these sections aren't fully consistent:
  - [ ] https://www.docsy.dev/docs/get-started/other-options/#prerequisites
  - [ ] https://www.docsy.dev/docs/get-started/docsy-as-module/installation-prerequisites/

**Comment by chalin** [comment]
Done as of 0.17.0:

- [Installation prerequisites](https://www.docsy.dev/docs/get-started/docsy-as-module/installation-prerequisites/) is the single home for prerequisite setup (#2672, 0.16.0).
- The [other-options prerequisites](https://www.docsy.dev/docs/get-started/other-options/#prerequisites) section cross-links it instead of duplicating (#2749, 0.17.0).
- Remaining polish (mode-neutral location, page naming) is tracked in #2676.

Closing as completed.
